### ISSUE TITLE
Wrong redirect

### ISSUE THREAD

**Issue by rodrigoaguilera** [idlewan/scribe-rip-redirector#1]
I just installed the extension and trying my first link
```
https://medium.com/@ruchidas28/4-books-that-saved-me-a-lot-of-money-on-online-courses-1ac938ebe48c?source=email-7d8fe1903c00-1656035439481-digest.weekly-83a207239c74-1ac938ebe48c----1-73------------------0e74276c_ef63_4d8c_8d0d_4fd409c4c755-1
```
and it doesn't redirect properly.

if I just replace `medium.com` with `scribe.rip` it works fine

**Comment by idlewan** [comment]
Thanks for the bug report, I have implemented with `v1.1` a better way to detect on which urls the redirect should work, which should make the extension useful again and fixes the issue.

**Comment by rodrigoaguilera** [comment]
Thanks!
I can confirm that with 1.1 the redirect happens properly :)
